scim/bulkRecords.bal

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
+
```

Comment:
Why is the license year not 2023? Check other places aswell

Reply:
ack